src/rgw/rgw_rest_s3.cc

```diff
@@ -3030,7 +3197,10 @@ RGWOp *RGWHandler_REST_Obj_S3::op_put()
 {
   if (is_acl_op()) {
     return new RGWPutACLs_ObjStore_S3;
+  } else if (is_tagging_op()) {
```

Comment:
For this sort of thing where we're following the style of a file to clean it up in one swell foop PR, just to avoid having a file that isn't internally consistent

(Some people might object to that on backport concerns. Maybe wait until some OTHER breaking change would make backports difficult anyway.)

Though all the permission checking now might be a good excuse to do a cleanup like that in rgw_op.cc